src/components/safe-apps/AppCard/index.tsx

```diff
@@ -92,10 +94,18 @@ const AppCardContainer = ({ url, children, variant }: AppCardContainerProps): Re
   )
 }
 
-const CompactAppCard = ({ url, safeApp, onPin, pinned }: CompactSafeAppCardProps): ReactElement => (
+const CompactAppCard = ({ url, safeApp, onPin, pinned, shareUrl }: CompactSafeAppCardProps): ReactElement => (
   <AppCardContainer url={url} variant="compact">
     <div className={styles.compactCardContainer}>
       <img src={safeApp.iconUrl} style={{ width: 52, height: 52 }} alt={`${safeApp.name} logo`} />
+      <CopyButton
+        text={shareUrl}
+        ariaLabel={`Share ${safeApp.name}`}
```

Comment:
Does the `ariaLabel` have to be different than the text? I would remove this prop and just use `initialToolTipText` for both.